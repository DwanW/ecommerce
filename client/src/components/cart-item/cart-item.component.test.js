import React from 'react';
import { shallow } from 'enzyme';
import CartItem from './cart-item.component';

describe('CartItem component test', () => {
    it('should render cartItem component', ()=>{
        const mockItem = {
            name:'belt',
            imageUrl:'www.testImage.com',
            price: 30,
            quantity: 50
        };
        expect(shallow(<CartItem item={mockItem} />)).toMatchSnapshot();
    })
})