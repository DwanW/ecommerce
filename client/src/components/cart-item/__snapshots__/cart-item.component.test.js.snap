// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CartItem component test should render cartItem component 1`] = `ShallowWrapper {}`;
